.github/workflows/test.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
         # When updating the minimum Python version here, also update the
         # `python_requires` from `setup.cfg`.
         # Run on latest minor release of each major python version.
-        python-version: [3.6, 3.7, 3.8, 3.9, 3.10rc1]
+        python-version: [3.6, 3.7, 3.8, 3.9, 3.10.0-rc.1]
```

Comment:
can we just set 3.10 and have it automatically updated just like the other version? Just asking.

Reply:
@altendky made the trio python-version auto pick the latest Python build, but I think it would be better to update it dependabot style